### DISCUSSION TITLE
ASReview Instruction Module

### DISCUSSION THREAD

**Discussion by Rinus-Verdonschot-UB** [asreview/asreview#1918] [Show and tell]
Dear forum members,

We have created an instruction module for ASReview (Oracle mode only) to be used our students/employees at Maastricht University (but feel free to distribute the link as our [skills modules](https://library.maastrichtuniversity.nl/teach/information-literacy/skills-modules/) are all Open Educational Resources).

[https://maastrichtuniversity.libwizard.com/f/SR_ASReview](https://maastrichtuniversity.libwizard.com/f/SR_ASReview)

We would be grateful for any tips, errors spotted, additions, etc. you could provide to improve this module, so please [let me know](mailto:rinus.verdonschot@maastrichtuniversity.nl) if you have any comments. 

Kind regards,

Rinus Verdonschot
University Library, Maastricht University

**Comment by Rensvandeschoot** [comment]
Great work! You might want to embed animations from our YouTube channel: https://www.youtube.com/channel/@ASReview
and maybe refer to the online teaching materials for more exercises: https://asreview.github.io/asreview-academy/

**Reply by Rinus-Verdonschot-UB** [reply]
Dear Rens, Thanks! Ill see if I can embed those into the module! Kind regards, Rinus
